releasenotes/notes/0.19/add-support-to-disable-amplitude-limit-in-parametric-pulses-ef88b77db8c1b06c.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Added an argument ``limit_amplitude`` to the constructor of ``ParametricPulse``,
+    which is the base class of :obj:`.Gaussian`, :obj:`.GaussianSquare`, :obj:`.Drag` and :obj:`.Constant`, to
+    enable or disable the amplitude limit of 1 on the pulse. This
+    option gets propagated to the parent class ``Pulse``. With this option enabled,
+    the ``ParametricPulse`` class and its subclasses do not allow amplitudes larger
+    than 1. By default, the value of ``limit_amplitude`` is true, so that the
+    amplitude is constrained to be less than 1. See `#6544 <https://github.com/Qiskit/qiskit-terra/issues/6544>`__ for more detail.
```

Comment:
Should this be an upgrade note as it's potentially a breaking change if someone was constructing a parametric pulse with an amplitude >1 prior to 0.19 it would have worked and now it will error unless they manually change the constructor call to set `limit_amplitude` false

Reply:
Looking at the history and the code, it's actually that `limit_amplitude` has always existed on the base class as a class attribute, it's just that this PR added an argument so you can _disable_ it on an instance-by-instance basis.  Have rewritten the note to reflect that.